### ISSUE TITLE
WCAG 2.1 compliance.

### ISSUE THREAD

**Issue by cdcavell** [cdcavell/cdcavell.name#270]
Links that open in new tabs/windows should be tagged for assistive technology
Requirement: Links that open in a new tab or a new window should either have an "aria-label" attribute or a screen-reader only element explaining to screen-readers that this opens in a new tab.

Keyboard focus should have a noticeable outline
Requirement: Interactive elements that can be navigated using the keyboard should be surrounded by a visual outline whenever they are focused.

Every page should include hidden links that allow to skip blocks
Requirement: Every page should include hidden links that by clicking on them (either using keyboard navigation or a screen-reader), the user will "skip" certain blocks directly to main landmarks such as main content, menu or footer.

Background images behaving as standard images should be tagged and described
Requirement: Background images that are not just for decoration purposes and should have the same treatment as standard images and include a "role" attribute that equals to "img" and an alternative text description in an "aria-label" attribute.

Page landmarks should be tagged and described for assistive technology
Requirement: Landmarks such as main content and footer should be built using their corresponding HTML5 element or include a description using an "aria-label" attribute and a "role" tag that equals to "contentinfo" or "main". Landmarks such as search and navigation are tested in other sections.

Font sizes should be big enough to be readable
Requirement: Font sizes should be at least 11px in size in order to stay readable in the majority of font families. This should be verified also when using dynamic units such as REM or percents.

Foreground and background colors have sufficient contrast
Requirement: Elements that have texts should meet a minimum contrast ratio of 4.5:1 between their foreground (usually text color) and background color.

[Web Accessibility - WCAG &amp; ADA Testing Tool - aCe](https://ace.accessibe.com/?utm_source=o&utm_medium=e&utm_campaign=u)
